pydantic/schema.py

```diff
@@ -635,9 +641,15 @@ def field_singleton_sub_fields_schema(
     (time, {'type': 'string', 'format': 'time'}),
     (timedelta, {'type': 'number', 'format': 'time-delta'}),
     (Json, {'type': 'string', 'format': 'json-string'}),
-    (IPvAnyAddress, {'type': 'string', 'format': 'ipvanyaddress'}),
-    (IPvAnyInterface, {'type': 'string', 'format': 'ipvanyinterface'}),
+    (IPv4Network, {'type': 'string', 'format': 'ipv4network'}),
+    (IPv6Network, {'type': 'string', 'format': 'ipv6network'}),
     (IPvAnyNetwork, {'type': 'string', 'format': 'ipvanynetwork'}),
+    (IPv4Interface, {'type': 'string', 'format': 'ipv4interface'}),
+    (IPv6Interface, {'type': 'string', 'format': 'ipv6interface'}),
+    (IPvAnyInterface, {'type': 'string', 'format': 'ipvanyinterface'}),
+    (IPv4Address, {'type': 'string', 'format': 'ipv4address'}),
+    (IPv6Address, {'type': 'string', 'format': 'ipv6address'}),
```

Comment:
I just [realized](https://github.com/json-schema-org/json-schema-spec/issues/540#issuecomment-494217561) that there's already an official/defined JSON Schema "format" for IPv4 and IPv6.

@euri10 can you change these format strings?:

`ipv4address` -> `ipv4`
`ipv6address` -> `ipv6`